test/automigrate.test.js

```diff
@@ -20,15 +20,13 @@ describe('cloudant automigrate', function() {
     Bar = db.define('Bar', {
       name: {type: String},
     });
-    db.once('connected', function() {
```

Comment:
Why are we removing `db.once('connected', cb)`? I thought it is needed to wait for the connection pool to be established.